tsconfig.json

```diff
@@ -14,8 +14,8 @@
     "removeComments": true,
     "sourceMap": true,
     "strict": true,
-    "lib": ["ES2019"],
-    "target": "ES2019"
+    "lib": ["ES2020"],
+    "target": "ES2020"
```

Comment:
This adds support for BigInt literals. We could also do `BigInt(...)` and stick with `ES2019` if that is preferred.